### ISSUE TITLE
Revise the wording on the "Congrats on completing the setup for Site Kit!" banner

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#9001]
Consider revising the wording on the "Congrats on completing the setup for Site Kit!" banner. At present it reads as:
"_Connect more services to see more stats_" 

This could be improved based on the following:
1. All services/modules may already be connected
2. "Stats" do not appear for Tag Manager and potentially future modules. 

![image](https://github.com/google/site-kit-wp/assets/41326532/47a479b2-7506-4113-971e-c151bb761896)

Consider the following change or similar:
```diff
- Connect more services to see more stats.
+ Connect more services or edit services from the settings screen.
```

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* [ ] 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by jimmymadon** [comment]
@marrrmarrr Very minor suggestion here but since this is such a popular banner, just wanted to ensure we say the right thing.

@jamesozzie 's points are valid - although, there is a very small chance that ALL services are connected when this banner is shown to users. (I think this might only happen if all services were connected already but a permissions error meant that they had to Redo setup).

And yes, not all Modules / Services show "stats". So should we change the wording. I would just add the word "connected" to @jamesozzie 's suggestion:

```diff
- Connect more services to see more stats.
+ Connect more services or edit connected services from the settings screen.
```

**Comment by marrrmarrr** [comment]
@jamesozzie @jamesozzie this update sounds good. Would it be possible in the case where all possible services are already connected to link instead to the "Connected Services" tab, instead of the other one?

**Comment by jimmymadon** [comment]
> this update sounds good. Would it be possible in the case where all possible services are already connected to link instead to the "Connected Services" tab, instead of the other one?

@marrrmarrr Yes, it is possible. However, I wouldn't bother too much about changing the functionality here since this would be a very very rare case (of setting up Site Kit with 0 inactive modules remaining). Also, the Setup Flow Refresh epic will change this banner anyway - so no point in spending too much time on this.

**Comment by eugene-manuilov** [comment]
AC + IB ✔️

**Comment by wpdarren** [comment]
### QA Update: ✅ 

Verified:

- On the  "Congrats on completing the setup for Site Kit!" banner I see the copy as per the AC. 

<img width="3274" height="594" alt="Image" src="https://github.com/user-attachments/assets/e4286c08-83c4-49dd-a494-d8a836a552fa" />
